deploy/003_deploy_optimism_spokepool.ts

```diff
@@ -9,7 +9,9 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // with deprecated spoke pool.
   // Set hub pool as cross domain admin since it delegatecalls the Adapter logic.
   const constructorArgs = [1_000_000, hubPool.address, hubPool.address];
```

Comment:
It's a bit confusing with 2x `constructorArgs` in the call to `deployNewProxy()`. Thoughts on renaming this initializer args?

```suggestion
  const initArgs = [1_000_000, hubPool.address, hubPool.address];
```